src/components/AggregationPanel/AggregationPanel.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {
+  TextElement,
+  KeyValueElement,
+  TableElement,
+  ImageElement,
+  VideoElement,
+  AudioElement,
+  ButtonElement,
+} from './AggregationPanelComponents';
+import styles from './AggregationPanel.scss';
+
+const BeatLoader = () => {
+  return (
+    <div className={styles.beatloader}>
+      <div className={styles.beat}></div>
+      <div className={styles.beat}></div>
+      <div className={styles.beat}></div>
+    </div>
+  );
+};
+
+const AggregationPanel = ({ data , isLoadingCloudFunction}) => {
+  return (
+    <>
+      { isLoadingCloudFunction ? <BeatLoader /> : Object.keys(data).length !== 0 ? (
```

Comment:
How about using existing `LoaderDots` or `Loader` component instead? I think `LoaderDots` should be a better fit here.

Reply:
Whats the difference?

Reply:
LoaderDots seem a better fit, they are less visually intrusive. The Loader balls flying around are a visual gimmick that we don't really want to use except maybe once on initial dashboard launch. It's like a splash screen of a mobile app.

Reply:
oky sure i will use `LoaderDots ` instead of introducing new component